src/ray/rpc/worker/direct_actor_client.h

```diff
@@ -115,14 +116,15 @@ class DirectActorClient : public std::enable_shared_from_this<DirectActorClient>
   ClientCallManager &client_call_manager_;
 
   /// Queue of requests to send.
-  std::deque<std::pair<std::unique_ptr<PushTaskRequest>, ClientCallback<PushTaskReply>>>
-      send_queue_;
+  GUARDED_BY(mutex_)
+  std::deque<std::pair<std::unique_ptr<PushTaskRequest>,
+                       ClientCallback<PushTaskReply>>> send_queue_;
 
   /// The number of bytes currently in flight.
-  int64_t rpc_bytes_in_flight_ = 0;
+  GUARDED_BY(mutex_) int64_t rpc_bytes_in_flight_ = 0;
```

Comment:
I think it's typically written `int64_t rpc_bytes_in_flight_ = 0 GUARDED_BY(mutex_);

Reply:
This didn't work for me because of the initialization, I think. I'll move the initialization into the constructor.